Essentials/src/main/java/com/earth2me/essentials/Settings.java

```diff
@@ -1948,4 +1962,39 @@ public boolean showZeroBaltop() {
     public int getMaxItemLore() {
         return config.getInt("max-itemlore-lines", 10);
     }
+
+    @Override
+    public Tag getPrimaryColor() {
+        return primaryColor;
+    }
+
+    private Tag _getPrimaryColor() {
+        final String color = config.getString("primary-color", "6");
```

Comment:
```suggestion
        final String color = config.getString("message-colors.primary", "6");
```
We may want to just use hex codes?